packages/tpetra/core/test/ImportExport2/ImportExport2_UnitTests.cpp

```diff
@@ -2026,7 +2026,7 @@ TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL( Import_Util, UnpackAndCombineWithOwningPIDs,
   RCP<const MapType> MapSource, MapTarget;
 
   const bool debug = ::Tpetra::Details::Behavior::debug ();
-  const bool verbose = ::Tpetra::Details::Behavior::verbose ();
+  verbose = ::Tpetra::Details::Behavior::verbose ();
```

Comment:
This doesn't quite make sense -- did the unit test capture a `verbose` variable from outside?

Reply:
Removes a shadow warning from https://github.com/trilinos/Trilinos/blob/e000e39a04969848a2e462def7dbde9e83f822ff/packages/tpetra/core/test/ImportExport2/ImportExport2_UnitTests.cpp#L117

Reply:
@kddevin Thanks for explaining!  I would have somewhat preferred getting rid of the outer `verbose` variable, but the way you did it is fine.  Thanks!